docker-hub/download-rate-limit.md

```diff
@@ -84,13 +84,13 @@ This means my limit is 100 pulls per 21600 seconds (6 hours), and I have 76 pull
 
 ### I don't see any RateLimit headers
 
-If you do not see these headers, that means pulling that image would not count towards pull limits. This could be because the image or your IP is unlimited in partnership with a publisher, provider, or an open-source organization.
+This could be because the image or your IP is unlimited in partnership with a publisher, provider, or an open-source organization. Pulling that image won’t count toward pull limits if you don’t see these headers. However, paid users pulling more than 5000 times daily require a Service Account subscription.
```

Comment:
@jerae-duffin Suggest "..users with a paid Docker subscription" instead of "paid users"

Could you also add a link to the Service Account page?